migrate/splunk.go

```diff
@@ -334,7 +336,13 @@ func splunkJob(cfgName string, progress SplunkToGravwell, cfg *cfgType, ctx cont
 	var count uint64
 	var byteTotal uint64
 
+	// we also maintain a moving average of the entry size
+	// we'll just initialize the average to something moderate
+	avgSize := float64(128)
```

Comment:
This initializes the exponential moving average of entry sizes. The initial number is almost irrelevant, it'll start to change as soon as we see actual entries, but I didn't want to start it at zero.

The alpha value is just a parameter of the moving average calculation; the higher alpha is, the stronger the effect of new values on the average.

I could break these out but they'll only be used here, I don't intend to make them user-adjustable, and as I said the values don't even matter particularly much -- I don't see much reason to hide the actual numbers at the top of the file instead of in the one place they're used.